lib/manager/pub/index.ts

```diff
@@ -1,8 +1,11 @@
+import { ProgrammingLanguage } from '../../constants';
 import { DartDatasource } from '../../datasource/dart';
 import * as npmVersioning from '../../versioning/npm';
 
 export { extractPackageFile } from './extract';
 
+export const language = ProgrammingLanguage.Dart;
```

Comment:
I updated this assuming that this is required for multiple managers using the same language

Reply:
I'd like to remove this, see other comment.